flink-table-store-benchmark/src/main/java/org/apache/flink/table/store/benchmark/metric/BenchmarkMetric.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.store.benchmark.metric;
+
+import org.apache.flink.table.store.benchmark.utils.BenchmarkUtils;
+
+import javax.annotation.Nullable;
+
+import java.util.Objects;
+
+/**
+ * Metric collected per {@link
+ * org.apache.flink.table.store.benchmark.BenchmarkOptions#METRIC_MONITOR_DURATION} for a single
+ * query.
+ */
+public class BenchmarkMetric {
```

Comment:
Should we add `copied from nexmark` for classes?

Reply:
No need. I've asked @wuchong and he says that we just need to add `inspired by Nexmark` in `README.md`.